commercial-schedules/src/main/java/org/dcsa/conformance/standards/cs/model/CsDateUtils.java

```diff
@@ -0,0 +1,122 @@
+package org.dcsa.conformance.standards.cs.model;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+
+public class CsDateUtils {
+  public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
The SimpleDateFormat is not synchronized or thread safe. Therefore, we should not make it static and reuse it. It can lead to data corruption if two threads are using it at the same time.